packages/framework/azure-client/src/interfaces.ts

```diff
@@ -30,6 +30,15 @@ export interface AzureClientProps {
     readonly logger?: ITelemetryBaseLogger,
 }
 
+export interface AzureContainerVersion {
+    // Version ID
+    id: string;
+
+    // Time when version was generated.
+    // ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ
+    date?: string;
```

Comment:
Why is the date optional?

Reply:
Existing storage API may not return us the date. It's an optional on `IVersion`.